README.md

```diff
@@ -1,11 +1,35 @@
-# Install tektoncd tools with brews
+# Install tektoncd Tools with Homebrew
 
-It is that simple :
+Homebrew formulas for working with Tekton!  
 
-```shell
+### Getting Started:
+
+For all formulas, start by running the command below:
+
+```
 brew tap tektoncd/tools
-brew install tektoncd-cli
```

Comment:
I like the extra details, but I found it useful to have a two liners to copy/paste to get started quickly.
Perhaps we can still have two lines in the getting started and then have sections for extra info?